todo_app/tests/test_integration.py

```diff
@@ -0,0 +1,157 @@
+import pytest, os
+from dotenv import find_dotenv, load_dotenv
+import todo_app.app as app
+from unittest.mock import patch, Mock
+
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    test_app = app.create_app()
+    with test_app.test_client() as client: 
+        yield client
+
+@patch('requests.get')
+def test_index_page(mock_get, client):
+    mock_get.side_effect = mock_get_cards
+    response = client.get('/')
+    print(response.data.decode())
```

Comment:
Once your test is passing we should remove this print

Reply:
done, thanks